package-lock.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "example-saas-contact-form",
+  "name": "@team-plain/example-next-advanced-contact-form",
```

Comment:
Can we do `@team-plain/example-nextjs-advanced-contact-form` (missing `js`) to make it consistent with the URL